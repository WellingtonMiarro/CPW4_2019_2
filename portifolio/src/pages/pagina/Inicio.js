import React, { Component } from 'react';

import Menu from '../../components/menu/Menu';
import './Estilo.scss';

export default class Inicio extends Component {
    render() {
        return (
            <div>
                <Menu inicio="/" bio="/bio"  contato="/contato"/>

                <div className="main">

                         <h1>Bem vindo ao meu portf√≥lio</h1>

                         <p>
                         There are many variations of passages of Lorem Ipsum available, but the majority
                         have suffered alteration in some form,
                         by injected humour, or randomised words which don't look even slightly believable.
                         </p>

                </div>
             
            </div>
        );
    }
}