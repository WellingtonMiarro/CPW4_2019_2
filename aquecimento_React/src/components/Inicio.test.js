import React from 'react';
import Inicio from './Inicio';

import toJson from 'enzyme-to-json';
//ele torna o snapshot ja em formato de texto
import {createSerializer} from 'enzyme-to-json';


// Utilizando para configuração do ENZYME
import { shallow, configure} from 'enzyme';
import  Adapter from 'enzyme-adapter-react-16';


configure({adapter: new Adapter()});
 expect.addSnapshotSerializer(createSerializer({mode:'deep'}));


describe("Teste de renderização do camponente INICIO",()=>{
    it("Verificar se a página Inicial e carregada adequadamente", ()=>{
        const wrapper = shallow(<Inicio/>);//retorna Serializado
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});