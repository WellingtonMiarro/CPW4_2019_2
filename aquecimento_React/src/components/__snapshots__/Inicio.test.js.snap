// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de renderização do camponente INICIO Verificar se a página Inicial e carregada adequadamente 1`] = `
<div>
  <h3>
    Bem vindo ao app de aquecimento!
  </h3>
  <p>
    Neste app, podemos ter um exemplo de como criar um app React básico.
  </p>
</div>
`;
